### ISSUE TITLE
How many epochs and what is the best loss that you trained your model?

### ISSUE THREAD

**Issue by tannq-jo** [UniModal4Reasoning/ChartVLM#12]
Hi,
I am trying to replicate the training procedure on ChartQA, PlotQA, Chart2Text, SimChart9K as described in your paper. It's my first time training such a big model so I don't know when I should stop the training loop. Could you please kindly provide the information about the number of epochs and the best loss that you trained your model with these 4 datasets.

Thank you for your attention to this matter.

**Comment by renqiux0302** [comment]
Hi @nguyenquangtan,
We have trained 50 and 5 epochs for the base decoder and auxiliary decoder respectively.

**Comment by tannq-jo** [comment]
Thank you for your information. May I ask what the best losses of both models are? I saw that you use the Cross-entropy loss. As our training data is different, the information about the best losses is really helpful for us to somehow determine the upper bound, therefore, knowing when is the right time to stop the training loop.

**Comment by Road2Redemption** [comment]
I also want to know the losses of both models. Have you guys figured it out? Thanks!
I tried to train several epochs but the result were bad :(
